### PR TITLE
Title section as 'New user?'

### PR DESCRIPTION
**Pull request by paramat**
For #129 
Don't use the words 'trying out', this understates MT as some kind of 'test' :D
Don't thank for downloading before the visitor downloads, this looks awkward.

### CODE REVIEW

File: downloads.html [hunk 1]
```diff
@@ -10,11 +10,9 @@
   - /other_distros/
 ---
 
-
-<h2>Thank you for trying out Minetest</h2>
+<h2>New user?</h2>
```
Comment (Ezhh) [inline downloads.html]: As a heading this really lacks context. I'd much prefer to see something like "Need help getting started?"

Comment (paramat) [reply downloads.html]: I'm open to a better title certainly.

**Comment by paramat** [comment]
I don't know why this section advises looking at the changelog, seems unrelated to new users and downloading. The changelog is complex dev stuff, not for new users.
Anyone know why? I might remove that.

File: downloads.html (hunk 1 above)
Comment (SmallJoker) [reply downloads.html]: `Getting started` sounds good, just like the MT wiki article name.

Comment (paramat) [reply downloads.html]: 'Getting started' seems ok to me, will do.

**Comment by paramat** [comment]
Updated as requested.
Note the changelog link has changed to 'https' hope that's ok.

**Pull request merged**